client/state/selectors/is-mapped-domain-site.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getRawSite } from 'state/sites/selectors';
+
+/**
+ * Returns true if site is a mapped domain site, false if the site is not,
+ * or null if the site is unknown.
+ *
+ * @param  {Object}   state  Global state tree
+ * @param  {Number}   siteId Site ID
+ * @return {?Boolean}        Whether site is a mapped domain site
```

Comment:
👍  Didn't realize others were using the white space for alignment. Not a big deal either way.